kimchi/src/circuits/polynomials/keccak/witness.rs

```diff
@@ -355,16 +355,25 @@ impl Chi {
 /// Values involved in Iota permutation step
 pub struct Iota {
     state_g: Vec<u64>,
+    rc: Vec<u64>,
```

Comment:
Optim: as it is fixed, we could have it static somewhere. It will be in the binary directly, and not in memory.

Reply:
Open https://github.com/o1-labs/proof-systems/issues/1605

Reply:
https://github.com/o1-labs/proof-systems/issues/1606